pytorch_lightning/trainer/trainer.py

```diff
@@ -765,7 +786,59 @@ def test(
             results = self.__test_using_best_weights(ckpt_path, test_dataloaders)
 
         self.teardown('test')
+        self.running_stage = RunningStage.UNDEFINED
+
+        return results
+
+    def predict(
+        self,
+        model: Optional[LightningModule] = None,
+        dataloaders: Optional[Union[DataLoader, List[DataLoader]]] = None,
```

Comment:
Any plans for direct support for datamodule?
(I know user can just do `dataloaders=dm.test_dataloader()`)

Reply:
DataModule won't really make sense as we plan to move closer to raw_data at some point.